web/cobrands/greenwich/base.scss

```diff
@@ -137,7 +127,7 @@ form {
                         text-decoration: underline;
                     }
                     &:visited {
-                        color: $links_visited;
+                        color: darken($link-color, 10%);
```

Comment:
I added this bit, it wasn't compiling the CSS because there was a remanent of `links_visited`

Reply:
I don't think this ever happens at all, in fact - I can't see a fake-link within a #form_sign_in within a fieldset anywhere.

Reply:
To be fair is not even needed. I just deleted in another commit

Reply:
Sorry, I meant that entire block is never used, the whole thing. I've removed it.